### DISCUSSION TITLE
Error: No connection could be made because the target machine actively refused it, director seems to be down or blocking our request.

### DISCUSSION THREAD

**Discussion by jxs714** [bareos/bareos#2089] [Q&A]
Hey guys,

Did a fresh install on windows 11. I setup the entire setup for it. I tried logging into the 127.0.0.1:9100 portal with admin/admin and another login I thought I created when you install and its giving this error. Is the port blocked or something is misconfigured? I know my ISP doesn't allow certain ports, but I use tailscale and its installed on windows but im still having this issue?

Any advice? It sounds like postgresql isnt working for some reason

**Comment by bruno-at-bareos** [comment]
This is not the appropriate place, you should use the user mailing list to get first help.
Discussion here are used to discuss development or enhancement of the product.

For your case, if postgresql is not running the chance the director is running are null and as such the web interface as no chance to connect.
